commands/query_completions.py

```diff
@@ -46,6 +46,7 @@ def return_completions(self, view, prefix, locations):
         # workspace = get_setting(SettingObject.workspace)
         open_tab = view.file_name()
         index_file = get_index_file(open_tab)
+        print(view, prefix, index_file)

```

Comment:
Is this `print` statement really needed? The same about other places where prints are being added.

Reply:
I'll get rid of it.